src/main/java/com/cisco/trex/stateless/model/RPCResponseError.java

```diff
@@ -0,0 +1,68 @@
+package com.cisco.trex.stateless.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RPCResponseError {
```

Comment:
Add @JsonIgnoreProperties(ignoreUnknown = true) to avoid exceptions if error model will be changed not critical.